src/fairseq2/models/llama/factory.py

```diff
@@ -72,6 +72,9 @@ class LLaMAConfig:
     dropout_p: float = 0.1
     """The dropout probability on outputs of Transformer layers."""
 
+    ffn_inner_dim_scale: float = 2 / 3
```

Comment:
Very nit: Can you please move this on top of `ffn_inner_dim_to_multiple` to group FFN related fields together?

Reply:
done